### ISSUE TITLE
a few questions

### ISSUE THREAD

**Issue by dmshel** [bonesoul/CoiniumServ#997]
There are a couple of ideas how to improve your project, is it possible to implement it?
1. Add the admin panel ( i see at config files AdminLTE this is admin panel?)
2. Add the ability to use different payment methods for workers simultaneously: Solo, PPS, PPLNS
3. Add Profit Switching statistic
4. autorefresh, graphs, etc.
Ищу кто сделает....

**Comment by BenKnigge** [comment]
Yes it's possible. Feel free to work on it. AdminLTE is the bootstrap theme.

**Comment by ultra-pool** [comment]
Has anyone made a basic admin portal yet?
